packages/shared/util.ts

```diff
@@ -6,6 +6,10 @@ export function truncateText(text: string, max_len: number) {
   }
 }
 
+/**
+ * @returns Whether the `url` is a contact or a group invite link.
```

Comment:
wording confusing, first time I read it I thought it means that the function returns "group" or "contact"

```suggestion
 * @returns Whether the `url` is an invite link (for contact or a group).
```